### PR TITLE
chore: change color of links 

### LINKED ISSUE #1309: [NG] Blogs - Author name
**Current Behavior**

The author names on individual blog pages are not being highlighted properly. 

**Desired Situation**

Add a uniform hover effect for author names (could be an underline, highlights, color transition to kepple green etc.)

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Tanuj22**
**Description**

This PR fixes #1309 
Also fixes some import issues and some css clean up

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
Kept the color of the link `keppel`. On hover, it's `charcoal`.

File: src/sections/Member-single/memberSingle.style.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ export const MemberSingleWrapper = styled.div`
             li {
                 a{
                     display: block;
+                    color: black;
```
Comment (Nikhil-Ladha) [inline src/sections/Member-single/memberSingle.style.js:100]: Isn't this by default?

Comment (Nikhil-Ladha) [reply src/sections/Member-single/memberSingle.style.js:100]: Ohh, sorry. I noticed it just now, that the default has changed 😅

**Review by Nikhil-Ladha — APPROVED** [review]
Now, since the color of the links are changed. I think other places will also be affected because of this..

Let's merge and see, if something is effected we will fix that as well.

**Comment by Nikhil-Ladha** [comment]
> Kept the color of the link `keppel`. On hover, it's `charcoal`.
> 
> For an example - https://deploy-preview-1311--layer5ng.netlify.app/blog/open-service-mesh/announcing-the-meshery-adapter-for-open-service-mesh

Is the text on the he site overall also, using charcoal?

**Comment by Nikhil-Ladha** [comment]
It doesn't seems notable 🤔

**Comment by Tanuj22** [comment]
The text is black

**Pull request merged**